src/components/button.scss

```diff
@@ -0,0 +1,219 @@
+@import "./base/config";
+
+/**
+ * @define default Button
+ */
+
+$Button-bg: $ui-white;
+$Button-bg-hover: $ui-background;
+$Button-color: $color-info;
+$Button-border-color: $ui-divider;
+$Button-border-color-hover: darken($Button-border-color, 10%);
+$Button-border-radius: $border-radius-medium;
+$Button-border-width: $border-width-small;
+$Button-border-bottom-width: $border-width-medium;
+$Button-disabled-opacity: 0.4;
+$Button-font-family: $base-font-family;
+$Button-font-size: $base-font-size;
+$Button-font-weight: $med-font-weight;
+$Button-height: 34px;
+$Button-line-height: $Button-height - 2;
+$Button-padding: 0 ($Button-height / 1.75);
```

Comment:
The `height`, `line-height` and `padding`, pulled from the current button, feel a little magic-numbery to me, but I was wanting to match the existing buttons exactly so there is no difference between the two if they are living side-by-side.